data.js

```diff
@@ -0,0 +1,36 @@
+export const petCareItemsArray = [
+    {
+		emoji: "🍖",
+		id: "food",
+		point: 1,
+	},
+    {
+		emoji: "⚽",
+		id: "play",
+		point: 2,
+	},
+    {
+		emoji: "🧼",
+		id: "groom",
+		point: 3,
+	},
+]
+
+export const achievementItemsArray = [
+    {
+        emoji: "🐕",
+        id: "dog",
+    },
+    {
+        emoji: "🐈",
+        id: "cat",
+    },
+    {
+        emoji: "💝",
+        id: "heart",
+    },
+    {
+        emoji: "🦴",
+        id: "bone",
+    },
+]
```

Comment:
We'll need to expand on this array. My suggestion below!

```suggestion
const achievementItemsArray = [
  {
    id: 1,
    name: "Welcome Home!",
    emoji: "🐾",
    condition: "Interact with your pet for the first time"
  },
  {
    id: 2,
    name: "Master Groomer",
    emoji: "🛁",
    condition: "Reach a certain number of grooming points"
  },
  {
    id: 3,
    name: "Gourmet Chef",
    emoji: "🥩",
    condition: "Reach X number of feeding points"
  },
  {
    id: 4,
    name: "Loving Companion",
    emoji: "⚽",
    condition: "Reach X number of playing points"
  },
  {
    id: 6,
    name: "Perfect Pooch",
    emoji: "🐶",
    condition: "Reach the maximum level for your dog"
  },
  {
    id: 7,
    name: "Cool Cat",
    emoji: "😺",
    condition: "Reach the maximum level for your cat"
  },
  {
    id: 10,
    name: "Loyal Friend",
    emoji: "💝",
    condition: "Interact with your pet for a certain number of consecutive days"
  }
];
```

Reply:
The "Loyal Friend" achievement is probably super hard to implement, so we can omit it